src/Microsoft.HttpRepl.Fakes/MockHttpMessageHandler.cs

```diff
@@ -19,8 +20,10 @@ public class MockHttpMessageHandler : HttpMessageHandler
         public MockHttpMessageHandler(IDictionary<string, string> urlsWithResponse,
             string header,
             bool readFromFile,
-            string fileContents)
+            string fileContents,
+            string contentType)
```

Comment:
It's a little odd to set the contentType on the MessageHandler, as that will apply to all messages from this handler.  However, since this is limited, it's probably fine - it'll only become a limitation when/if we have a test that has multiple responses with separate content types.  (At which point we might want to focus on the scope of the test being too large... maybe).  We'll cross that bridge when we come to it.

Reply:
Created an issue to clean this up:
https://github.com/aspnet/HttpRepl/issues/120